Cartography.xcodeproj/project.pbxproj

```diff
@@ -788,6 +788,7 @@
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				APPLICATION_EXTENSION_API_ONLY = YES;
+				CODE_SIGN_IDENTITY = "Mac Developer";
```

Comment:
No, originally you had "iPhone Developer" identity in the global project setting, which is then propagated to Mac Framework as well.
As for the platform setting, I'm aware of it, but thanks :)